### PR TITLE
Packaging and license

### PR DESCRIPTION
**Pull request by chrisburr**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (chaen) [inline LICENSE]: shall we apply it since we are at it ?

File: setup.py [hunk 2]
```diff
@@ -1,23 +1,38 @@
...
+    install_requires=[
+        'enum34; python_version < "3.4"',
+        'm2crypto',
+        'tornado',
+    ],
     packages=['tornado_m2crypto', 'tornado_m2crypto.test'],
-    **kwargs)
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'OSI Approved :: GNU General Public License v3 or later (GPLv3+)',
```
Comment (chaen) [inline setup.py:27]: Isn't that contradictory wit hthe Apache license above ?

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (chaen) [inline LICENSE]: I don't think we want that. DIRAC and LHCbDIRAC uses GPL3. So why Apache2 here ?

Comment (chrisburr) [reply LICENSE]: I had a `LICENSE.1` file sat in my local repository and committed the wrong one...

Fixed.

File: LICENSE (hunk 1 above)
Comment (fstagni) [reply LICENSE]: There's no name, right now. I guess that would be the "DIRAC consortium" as the only legal entity here.

Comment (chrisburr) [reply LICENSE]: The current file in this PR is the same as: https://github.com/DIRACGrid/DIRACOS2/blob/main/LICENSE

Is there something that should be changed about both?

Comment (fstagni) [reply LICENSE]: No, nothing to be changed for these licences.

**Review by chaen — APPROVED** [review]

**Pull request merged**